### ISSUE TITLE
ERR_INVALID_ARG_TYPE when running less task

### ISSUE THREAD

**Issue by landlockedsurfer** [sbt/sbt-less#105]
Hi!
I ran into an issue with `sbt-less` resulting in the following build error

```
[info] LESS compiling on 1 source(s)
[error] internal/fs/utils.js:701
[error]     throw new ERR_INVALID_ARG_TYPE(
[error]     ^
[error] 
[error] TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received undefined
[error]     at Object.writeFile (fs.js:1385:5)
[error]     at /Users/landlockedsurfer/src/trash/sbt-less-example/project/target/less/lessc.js:89:20
[error]     at /Users/landlockedsurfer/src/trash/sbt-less-example/project/target/node-modules/webjars/mkdirp/index.js:47:26
[error]     at FSReqCallback.oncomplete (fs.js:177:5) {
[error]   code: 'ERR_INVALID_ARG_TYPE'
[error] }
...
```

The build error popped up yesterday all of a sudden (most likely due to a refresh of a node dependency).
After creating a standalone project I noticed that that issue seems to be related to files that are starting with the definition of a `less` variable.
I could not track down the cause of the problem but found a workaround by adding some additional lines at the top of the file.
E.g.
This file causes a build error
```
// /* add multiline comment to make things work, i.e. remove double slash at the start of this line */
// The problem seems to occur whenever the file starts with a variable declaration
@white: #fff;
```
By enabling the multiline comment at the start the `less` task runs fine again. 
```
/* add multiline comment to make things work, i.e. remove double slash at the start of this line */
// The problem seems to occur whenever the file starts with a variable declaration
@white: #fff;
```
I've created a sample project to demonstrate the issue here: https://github.com/landlockedsurfer/sbt-less-test

Posting this workaround here in case someone else stumbles across this issue since I could not find anything in the internets.

**Comment by sbmpost** [comment]
might be related to newer version of node

**Comment by TheNorthMemory** [comment]
I've also got the similar `error` messages whose came from `lessc.js:89:20`.

https://github.com/sbt/sbt-less/blob/60a7e5367c16f88eacaf675afbb8f8117e46a752/src/main/resources/lessc.js#L87-L90

Checking the `fs.writeFile((file, data[, options], callback))` manual, it was changed the behavior since v14.0.0 with: 

> The data parameter won't coerce unsupported input to strings anymore.

In L69, there was checked the `content` but L89 didn't which should be `undefined` while `less.sourceMap` was turned as `false`. it's caused the `fs.writeFile` failed and then prompt above messages.
